lib/npm-installation-manager.ts

```diff
@@ -79,7 +79,7 @@ export class NpmInstallationManager implements INpmInstallationManager {
 		}
 
 		if (shouldInstall) {
-			await this.$childProcess.exec(`npm install ${inspectorNpmPackageName}@${version} --prefix ${cachePath}`);
+			await this.$childProcess.exec(`npm install ${inspectorNpmPackageName}@${version} --prefix ${cachePath}`, { maxBuffer: 250 * 1024 });
```

Comment:
the default value is 200*1024 , is it enough to just add 50 KB more?

Reply:
@rosen-vladimirov it seems to be enough for the installation to succeed.